asv/benchmarks.py

```diff
@@ -120,6 +120,10 @@ def log_result(msg):
         result['started_at'] = datetime.datetime.utcnow()
 
         for param_idx, params in param_iter:
+            if benchmark['params'] and param_idx not in benchmark['_selected_idx']:
+                bench_results.append(dict(samples=None, number=None, result='skipped', stats=None))
```

Comment:
`result=nan` is used for skipped values elsewhere, and can be used also here so that it's not necessary postprocess it out afterward or change printing.